modules/oe_theme_helper/config/schema/oe_theme_helper.schema.yml

```diff
@@ -0,0 +1,10 @@
+image.effect.retina_image_scale:
+  type: image_size
+  label: 'Retina Image scale'
```

Comment:
Also let's not call it "Retina" cause it has nothing to do with retina per se. It's a multiplier based. If I configure a multiplier of 10, it has nothing to do with retina. So I'd call the plugin `MultiplierImageScale`. Though it's also only upscales if the image is smaller than the configuration. Any ideas for naming?

Reply:
We could change the configuration form so that the users are forced to select certain multipliers. Currently only 2x and 3x retina screens are available (after a quick google search, let me know if there are more options) so we could restrict the field to those two options. Would this make sense?

Reply:
Yes, true, makes sense what you say.